HousingRegisterApi/V1/Services/BedroomCalculationService.cs

```diff
@@ -0,0 +1,230 @@
+using HousingRegisterApi.V1.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HousingRegisterApi.V1.Services
+{
+    public static class BedroomCalculationService
+    {
+        //public static int CalculateOrig(Application application)
```

Comment:
Remove commented code.

Reply:
yeah, just left it in there is Dan wanted to compare the old and new..  but obviously will remove if everything else is ok

Reply:
removed